app/page.tsx

```diff
@@ -180,63 +180,57 @@ export default function Page() {
         </Badge>
         .
       </p>
-      <div className="columns-2 sm:columns-3 gap-4 my-8">
-        <div className="relative h-40 mb-4">
+      <div className="grid grid-cols-2 grid-rows-4 sm:grid-rows-3 sm:grid-cols-3 gap-4 my-8">
+        <div className="relative h-40">
           <Image
             alt="Me speaking on stage at React Summit about the future of Next.js"
             src={summit}
             fill
-            sizes="(max-width: 768px) 213px, 33vw"
             priority
             className="rounded-lg object-cover"
           />
         </div>
-        <div className="relative h-80 mb-4 sm:mb-0">
-          <Image
-            alt="Me, Lydia, and Delba filming the Next.js Conf keynote"
-            src={filming}
-            fill
-            sizes="(max-width: 768px) 213px, 33vw"
-            priority
-            className="rounded-lg object-cover object-[-16px] sm:object-center"
-          />
-        </div>
-        <div className="relative h-40 sm:h-80 sm:mb-4">
+        <div className="relative sm:row-span-2 row-span-1">
           <Image
             alt="Me standing on stage at Reactathon delivering the keynote"
             src={reactathon}
             fill
-            sizes="(max-width: 768px) 213px, 33vw"
             priority
             className="rounded-lg object-cover object-top sm:object-center"
           />
         </div>
-        <div className="relative h-40 mb-4 sm:mb-0">
+        <div className="relative">
           <Image
-            alt="Me standing on stage at SmashingConf giving a talk about my optimism for the web"
-            src={smashing}
+            alt="Me and Guillermo Rauch on stage for Vercel Ship, answering questions from the Next.js community"
+            src={ship}
             fill
-            sizes="(max-width: 768px) 213px, 33vw"
```

Comment:
Is there a reason sizes was removed?

Reply:
Yes, I thought it could be redundant given the height, grid & row  attributes of the grid. Adding them gives no visual difference, so I am open to learn which way makes more sense

Reply:
I checked it again, and it seems images will be better optimized and have smaller file sizes, although the rendered sizes will be the same. I added it back to the code.